cpp/src/lists/contains.cu

```diff
@@ -51,29 +52,74 @@ auto get_pair_iterator(cudf::column_device_view const& d_search_keys)
   return d_search_keys.pair_begin<ElementType, has_nulls>();
 }
 
-template <typename ElementType, bool>
+template <typename ElementType,
+          bool,
+          std::enable_if_t<!cudf::is_fixed_point<ElementType>(), void>* = nullptr>
 auto get_pair_iterator(cudf::scalar const& search_key)
 {
   return cudf::detail::make_pair_iterator<ElementType>(search_key);
 }
 
+template <typename ElementType,
+          bool,
+          std::enable_if_t<cudf::is_fixed_point<ElementType>(), void>* = nullptr>
+auto get_pair_iterator(cudf::scalar const& search_key)
```

Comment:
Maybe call `make_pair_rep_iterator`? That sounds like something that would be useful to have in other places.

Reply:
Will do. I will have `lists::<unnamed>::get_pair_iterator()` call `cudf::detail::make_pair_rep_iterator()`.

Reply:
Done.